src/events.cpp

```diff
@@ -304,6 +303,16 @@ void Events::disconnectFilterSignals(obs_source_t *filter)
 	signal_handler_t *sh = obs_source_get_signal_handler(filter);
 	signal_handler_disconnect(sh, "enable", OnSourceFilterVisibilityChanged, this);
 }
+void Events::initialiseHotkeysHooks()
```

Comment:
you are already initializing hotkeys in the hotkey instance, why are you doing it again here?

Reply:
As described above - only at this point OBS Hotkeys are actually available and registered